index.html

```diff
@@ -2,174 +2,292 @@
-        <p>Hello! I am a software developer! I can help you build a product, feature or website. Take a look of my
-            works.
-            <br>If you like what you see and have a project you need coded, don't hesitate and contact me.
+  <header>
+    <nav class="navbar ">
+      <img class="icon" src="images/image_geometry_1.svg" alt="logo" />
+      <button class="nav-btn"></button>
+      <ul class="nav-list">
+        <li><a href="#hello"> Hello</a></li>
+        <li><a href="#projects"> Portfolio</a></li>
+        <li><a href="#about"> About</a></li>
+        <li><a href="#contact">Contact</a></li>
+      </ul>
+    </nav>
+
+    <nav class="open-nav disappear">
+      <div class="drawer-image"></div>
+      <div class="drawer-image-1"></div>
+      <div class="drawer-image-2"></div>
+      <div class="drawer-image-3"></div>
+      <button class="close-drawer-btn"> </button>
+      <ul class="drawer-menu-items">
+        <li><a href="#hello"> Hello</a></li>
+        <li><a href="#projects"> Portfolio</a></li>
+        <li><a href="#about"> About</a></li>
+        <li><a href="#contact">Contact</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <section class="intro-section" id="hello">
+      <article class="white-bg">
+        <h1 class="heading">Lekan Jimoh</h1>
+        <p class="para">
+          Hello! I am a software developer! I can help you build a product,
+          feature or website. Taka a look of my works. If you like what you
+          see and have a project you need coded, don't hesitate and contact
+          me.
+        </p>
+        <button class="buttons-primary-dark" type="button">
+          Start collaboration
+        </button>
+      </article>
+      <img class="page-brk-dp" src="images/image-geometry_1.svg" alt="page break" />
+    </section>
+    <img class="page-brk" src="images/image_geometry_2.svg" alt="page break" />
+    <section class="works" id="projects">
+
+      <h2 class="heading heading-works">Projects</h2>
+
+      <!-- start of additional code for desktop -->
+      <div class="project-card-desktop">
+
+        <!-- two elements inside the desktop projects -->
+        <div class="desktop-project-1">
+          <div class="d-p-1-details">
+            <h4>Your project name goes here</h4>
+            <ul>
+              <li>HTML/CSS</li>
+              <li>Ruby on Rails</li>
+              <li>JavaScript</li>
+            </ul>
+            <a href="#">See project <i class="fa fa-arrow-right"></i></a>
+          </div>
+        </div>
+        <div class="desktop-project-2">
+          <div class="d-p-2-details">
+            <h4>Project name goes...</h4>
+            <ul>
+              <li>HTML/CSS</li>
+              <li>Ruby on Rails</li>
+              <li>+1</li>
+            </ul>
+            <a href="#">See project <i class="fa fa-arrow-right"></i></a>
+          </div>
+          <img src="images/Rectangle21(4).png" alt="project-image" width="100">
+        </div>
```

Comment:
- [ ] It is better not use inline styles in your `HTML` file. We should keep your style definition in a separate CSS file. Please remove the inline style. :writing_hand: 

- [ ] Please add proper width and height for the image to match with the Figma design. :writing_hand: 

> - **Your Implementation:**
![p1](https://user-images.githubusercontent.com/70483835/154755625-0160b083-a0b9-4385-9eaf-14f3d0924e5f.png)

> - **Figma Design:**
![p2](https://user-images.githubusercontent.com/70483835/154755635-0f8a151a-eefb-4f55-80bc-c43ffaf92dfe.png)